symfony/src/Controller/ArticleController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Article;
+use Algolia\SearchBundle\IndexManagerInterface;
+use Symfony\Component\HttpFoundation\Response;
+
+class ArticleController
+{
+
+    protected $indexManager;
+
+    public function __construct(IndexManagerInterface $indexingManager)
+    {
+        $this->indexManager = $indexingManager;
+    }
+
+    public function index()
+    {
+        $articles = $this->indexManager->rawSearch('', Article::class);
+        var_dump($articles);
+        $response = new Response();
+        return $response;
+    }
+}
```

Comment:
Format the code before delivery please.

Reply:
I've used the command on phpStorm still get an issue on it apparently.